hybrid/SampleApps/SmartSyncExplorerHybrid/SmartSyncExplorerHybrid.xcodeproj/project.pbxproj

```diff
@@ -3,7 +3,7 @@
 	archiveVersion = 1;
 	classes = {
 	};
-	objectVersion = 49;
```

Comment:
👍   But do we know how this happens?  A few months ago this broke the SmartStore unit tests (because 49 doesn't exist) and I had to make this change.  Is it just a bug in xcode?

Reply:
That XCode did - I can revert it.